itl/plugins-contrib.d/systemd.conf

```diff
@@ -0,0 +1,54 @@
+/*
+ * Icinga2 CheckCommand definition for the `check_systemd.py` Monitoring Plugin
+ */
+object CheckCommand "systemd" {
+    import "plugin-check-command"
+
+    command = [ PluginContribDir + "/check_systemd.py" ]
+
+    arguments = {
+        "--unit" = {
+            value = "$systemd_unit$"
+            description = "Name of the systemd unit that is being tested."
+        }
+        "--exclude" = {
+            value = "$systemd_exclude_unit$"
+            description = "Exclude a systemd unit from the checks. This option can be applied multiple times. Also supports regular expressions."
+            repeat_key = true
+        }
+        "--no-startup-time" = {
+            set_if = "$systemd_no_startup_time$"
+            description = "Don’t check the startup time. Using this option the options `systemd_warning` and `systemd_critical` have no effect. (Default: `false`)"
+        }
+        "--warning" = {
+            value = "$systemd_warning$"
+            description = "Startup time in seconds to result in a warning status. (Default: `60s`)"
+        }
+        "--critical" = {
+            value = "$systemd_critical$"
+            description = "Startup time in seconds to result in a critical status. (Default: `120s`)"
+        }
+        "--dead-timers" = {
+            set_if = "$systemd_dead_timers$"
+            description = "Detect dead / inactive timers. (Default: `false`)"
+        }
+        "--dead-timers-warning" = {
+            value = "$systemd_dead_timers_warning$"
+            description = "Time ago in seconds for dead / inactive timers to trigger a warning state (by default 6 days)."
+        }
+        "--dead-timers-critical" = {
+            value = "$systemd_dead_timers_critical$"
+            description = "Time ago in seconds for dead / inactive timers to trigger a critical state (by default 7 days)."
+        }
+        "-v" = {
```

Comment:
According to the official documentation this parameter is not a boolean but a count data type and therefore it would be appropriate here to just use ``value`` instead of ``set_if``.